miden-tx/src/executor/mast_store.rs

```diff
@@ -47,6 +49,10 @@ impl TransactionMastStore {
         let miden_lib_forest = MidenLib::default().mast_forest().clone();
         store.insert(miden_lib_forest);
 
+        // load utils lib MAST forest
+        let utils_lib_forest = UtilsLib::default().mast_forest().clone();
+        store.insert(utils_lib_forest);
```

Comment:
I'm not sure I like that we now need to care about another library. Ideally, we'd just vendor the utils library into the `miden` library at compile time, but unfortunately, Miden assembler does not yet support vendoring. So, maybe trying the approach with a single MASM file is worth it.

cc @plafer, @bitwalker for another use case for vendored library.